internal/cmd/server.go

```diff
@@ -38,6 +42,10 @@ func getServerCommand() *cobra.Command {
 	}
 	cmd.PersistentFlags().StringVarP(&cfgPath, "config-path", "c", "",
 		"The path to the configuration file.")
+	cmd.PersistentFlags().BoolVarP(&development, "development", "d", true,
```

Comment:
this looks a bit abstract, maybe `log-development-mode` is better?
and you should mention it's somehting form zap log in description.